include/spral_ssids.h

```diff
@@ -56,6 +56,12 @@ struct spral_ssids_inform {
    char unused[76]; // Allow for future expansion
 };
 
+struct spral_numa_region {
```

Comment:
I guess this could be used for `spral::hw_topology::NumaRegion` too (guess_topology.hxx). Having this just once on the C side and the corresponding interoperable type on the Fortran side would be neat. But maybe it's simpler this way.

Reply:
Yeah I think it's clearer this way for the user as they can see the full struct.